### PR TITLE
#9: implement NavBar (closes #9)

### LINKED ISSUE #9: implement NavBar
<- #7 
## requirements
- left: the app name (GH-Explorer? you can choose!)
- right: nothing
- center: `SearchBar`
  - composed by one `input` and one `button`
  - should notify parent with new `value` when user clicks on button
- UI
  - content `max-width: 1000px`
  - you can style the rest as you wish (`background-color`, `height` ....)
## specs
- use `NavBar` (https://github.com/buildo/react-components/tree/master/src/nav-bar)

### PR DESCRIPTION
**Pull request by nerder**
Issue #9 
## Test Plan
- You will see a Logo on the left, clicking on it will return you to the homepage
- Right side should be clear as expected
- `SearchBar` will be centered
- Type something on the `SearchBar` with the console open you will see the output as a PoC
- The NavBar have `1000px` maxWidth as requested try zoom out and you will see it will not grow more than `1000px`
- I styled it using Material Design standards, check if it seems good for your

### CODE REVIEW

File: src/components/NavBar/nav-bar.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.nav-bar{
+    z-index: 9999;
```
Comment (FrancescoCioria) [inline src/components/NavBar/nav-bar.scss]: what for? It doesn't have `position: absolute/fixed` (right?)

File: src/components/SearchBar/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@skinnable()
+@props({
+  value: t.String,
+  onChange: t.Function
+})
+export default class SearchBar extends React.Component {
+
+  onSearch = () => {
+    const input = React.findDOMNode(this.refs.input);
+    this.props.onChange(input.value);
```
Comment (FrancescoCioria) [inline src/components/SearchBar/SearchBar.js:14]: nothing, talked with gio. ignore this comment

File: src/components/SearchBar/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    return {
+      value,
+      onSearch: this.onSearch
+    };
+  }
+
+  template({ value, onSearch }) {
+    return (
+      <div className="searchBar">
+        <input type="text" placeholder="Search your Repository" ref="input" defaultValue={value} required></input>
```
Comment (FrancescoCioria) [inline src/components/SearchBar/SearchBar.js:28]: EDIT: ignore this comment

File: src/components/NavBar/nav-bar.scss (hunk 1 above)
Comment (nerder) [inline src/components/NavBar/nav-bar.scss]: @FrancescoCioria 
actually nav-bar doesn't have any specified position, you want me to add `position:fixed` and make it sticky?

Comment (FrancescoCioria) [inline src/components/NavBar/nav-bar.scss]: no, it shouldn't be fixed (also, if you wanted it to be fixed you could simply pass `fixed={true}` to it ;) )
what I mean is: as it it doesn't have `position: absolute` nor `position: fixed` the property `z-index` is useless -> please remove it :)

**Comment by giogonzo** [comment]
except for a few indentation/style issues, it looks good to me :)

as for the style and transitions, not going to express my opinion about it :P (cool anyway)

notes:
- as discussed before, state handling and `onChange`/`value` props as done now are not fully working / too complex: opened #16 to track this

**Pull request merged**